src/Api/Function.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * @Author: Erik Slovak <erik.slovak@technologystudio.sk>
+ * @Date: 2023-08-15T18:08:23+02:00
+ * @Copyright: Technology Studio
+**/
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const isFunction = (value: any): value is (...args: any[]) => any => (
```

Comment:
any -> unknown ?

Reply:
can't we just use typeof something === function exactly in code, that should be cheaper than utility type

but if we would need one this one can be used instead 

```typescript
const isFunction = <T, F extends (...args: unknown[]) => unknown>(value: T): value is Extract<T, F> => {
    return typeof value === 'function';
}
```